tools/tpm2_encryptdecrypt.c

```diff
@@ -59,6 +59,23 @@ static tool_rc readpub(ESYS_CONTEXT *ectx, TPM2B_PUBLIC **public) {
                       public, NULL, NULL);
 }
 
+static void append_pkcs7_padding_data_to_input(uint8_t *pad_data) {
```

Comment:
you also need to strip on decrypt operation if -e is specified.

Reply:
Yep. That is already added to the to-do list #1547

Reply:
Done